src/iidm/Identifiable.cpp

```diff
@@ -118,7 +169,14 @@ void Identifiable::reduceVariantArraySize(unsigned long number) {
 
 void Identifiable::removeAlias(const std::string& alias) {
     getNetwork().getIndex().removeAlias(*this, alias);
-    m_aliases.erase(alias);
+    auto it = std::find_if(m_aliasesByType.begin(), m_aliasesByType.end(), [&alias](const std::pair<std::string, std::string>& entry) {
+        return entry.second == alias;
+    });
+    if (it != m_aliasesByType.end()) {
+        m_aliasesByType.erase(it);
+    } else {
+        m_aliasesWithoutType.erase(alias);
+    }
```

Comment:
Same remark

Reply:
It is not relevant in C++ because if the type is "", it is not in `m_aliasesByType`